gradle-sls-packaging/src/test/groovy/com/palantir/gradle/dist/service/JavaServiceDistributionPluginTests.groovy

```diff
@@ -1300,6 +1300,76 @@ class JavaServiceDistributionPluginTests extends GradleIntegrationSpec {
         actualStaticConfig.jvmOpts.containsAll(['-XX:+UseG1GC', '-XX:+UseNUMA', "-XX:MaxGCPauseMillis=500"])
```

Comment:
Should we update this test to use `javaVersion 17` to explicitly test JDK 17 and avoid breaking when default `javaVersion` for the project bumps to JDK 21?

Reply:
good idea, updated!